apps/backoffice-v2/src/Router/Router.tsx

```diff
@@ -21,35 +24,57 @@ const router = createBrowserRouter([
     loader: rootLoader,
     errorElement: <RootError />,
     children: [
-      ...(env.VITE_AUTH_ENABLED
-        ? [
-            {
-              path: '/:locale/auth/sign-in',
-              element: <SignIn />,
-            },
-          ]
-        : []),
       {
-        path: '/:locale',
-        element: <Locale />,
-        loader: localeLoader,
+        element: <UnauthenticatedLayout />,
```

Comment:
Protected and unprotected routes are now expected to be wrapped by their respective layouts instead of leaning on strings configured by the developer of the backoffice.